project.clj

```diff
@@ -29,6 +29,17 @@
                  [bigml/histogram "4.1.3"]                            ; Streaming one-pass Histogram data structure
                  [buddy/buddy-core "1.2.0"]                           ; various cryptograhpic functions
                  [buddy/buddy-sign "1.5.0"]                           ; JSON Web Tokens; High-Level message signing library
+                 [org.apache.hadoop/hadoop-common "2.7.3"]
+                 [org.apache.drill.exec/drill-jdbc-all "1.10.0"
+                  :exclusions [org.slf4j/log4j-over-slf4j
+                               org.slf4j/jcl-over-slf4j
+                               org.slf4j/slf4j-api
+                               log4j]]
+                 [org.spark-project.hive/hive-jdbc "1.2.1.spark2"
+                  :exclusions [org.codehaus.jackson/jackson-xc
+                               org.eclipse.jetty.aggregate/jetty-all
+                               org.mortbay.jetty/jetty]
+                  :classifier "standalone"]
```

Comment:
can you keep the dependencies in alphabetical order? That makes it easier to see what we're currently using/find one you're looking for

Reply:
also https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#dependencies-should-have-comment

Reply:
Done (passing eastwood)